cinema/serializers.py

```diff
@@ -22,3 +22,66 @@
         instance.save()
 
         return instance
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=255)
+
+    def create(self, validated_data):
+        name = validated_data.get("name")
+        if Genre.objects.filter(name=name).exists():
+            raise serializers.ValidationError("This name already exist")
```

Comment:
Typo in the error message: 'This name already exist' should be 'This name already exists'.